keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -1835,8 +1836,8 @@ For Alpha, describe what tests will be added to ensure proper quality of the enh
 For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
 https://storage.googleapis.com/k8s-triage/index.html
 -->
-
-- <test>: <link to test coverage>
+The integration test will be added to test the admission policy expression validation with feature gate on/off:
```

Comment:
integration tests? (plural)

Reply:
Also, the comment for this says "describe what tests will be added to ensure proper quality of the enhancement".

Reply:
Update with more details